<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceManagement</class>
 <widget class="QWidget" name="AttendanceManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attendance Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="attendanceTypeComboBox">
       <item>
        <property name="text">
         <string>Student</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teacher</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="searchBar">
     <property name="placeholderText">
      <string>Search by ID or Name...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="attendanceTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="markAttendanceButton">
     <property name="text">
      <string>Mark Attendance</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
